From c8d493074e2ef1dd43b5f7ab246566bb937b923f Mon Sep 17 00:00:00 2001
From: Allen Ye <allen.ye@mediatek.com>
Date: Tue, 12 Nov 2024 18:16:28 +0800
Subject: [PATCH 005/149] hostapd: Fix definition of 6GHz operating class 137

The channel sets should follow the op_class 131, and the inc should be
64 to fit the bandwidth 320MHz.

Signed-off-by: Allen Ye <allen.ye@mediatek.com>
---
 src/common/ieee802_11_common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common/ieee802_11_common.c b/src/common/ieee802_11_common.c
index 4a35479fc..26360a559 100644
--- a/src/common/ieee802_11_common.c
+++ b/src/common/ieee802_11_common.c
@@ -2462,7 +2462,7 @@ const struct oper_class_map global_op_class[] = {
 	{ HOSTAPD_MODE_IEEE80211A, 136, 2, 2, 4, BW20, NO_P2P_SUPP },
 
 	/* IEEE P802.11be/D5.0, Table E-4 (Global operating classes) */
-	{ HOSTAPD_MODE_IEEE80211A, 137, 31, 191, 32, BW320, NO_P2P_SUPP },
+	{ HOSTAPD_MODE_IEEE80211A, 137, 1, 233, 64, BW320, NO_P2P_SUPP },
 
 	/*
 	 * IEEE Std 802.11ad-2012 and P802.ay/D5.0 60 GHz operating classes.
-- 
2.45.2

